docs/content/how-to-guides/configure-reactions/configure-debezium-reaction/_index.md

```diff
@@ -3,8 +3,149 @@ type: "docs"
 title: "Configure a Debezium Reaction"
 linkTitle: "Configure a Debezium Reaction"
 weight: 40
-toc_hide: true
-hide_summary: true
 description: >
     Learn how to configure a Debezium Reaction
----
\ No newline at end of file
+---
+
+The Drasi Debezium Reaction connector generates a Debezium-compatible data change event for each Added, Updated, or Removed in a Drasi result for a given Continuous Query. The events produced by this Reaction should be analogous to the data change events produced by similar Debezium Connectors, which can be used as a reference model and more detailed commentary. 
+
+
+## Requirements
+On the computer from where you will create the Drasi Debezium Reaction, you need to install the following software:
+- [Drasi CLI](/reference/command-line-interface/) 
+
+## Creating the Reaction
+To create a Reaction, execute the `drasi apply` command as follows:
+
+```text
+drasi apply -f my-reaction.yaml -n drasi-namespace
+```
+
+The `drasi apply` command is how you create all new Drasi resources (in this case a Reaction). The `-f` flag specifies that the definition of the new Reaction is contained in the referenced YAML file `my-reaction.yaml` and the `-n` flag specifies the Drasi namespace in which to create the Reaction (Drasi must already be installed in that namespace).
+
+## Reaction Definitions
+The YAML file passed to `drasi apply` can contain one or more Reaction definitions. Here is an example of a Drasi Debezium Reaction definition:
+
+```yaml {#debezium}
+apiVersion: v1
+kind: Reaction
+name: debezium-reaction
+spec:
+  kind: Debezium
+  queries:
+    <query-id>:
+  properties:
+    brokers: <kafka-broker>
```

Comment:
How do you specify credentials if your kafka is password protected?

Reply:
Good point. I have added two properties: saslPassword and saslUsername. This enables users to use SASL/PLAIN to authenticate to the kafka producer